src/DynamoCoreWpf/NodeViewCustomization/NodeViewCustomizations/FunctionNodeViewCustomization.cs

```diff
@@ -59,8 +61,6 @@ public void CustomizeView(Function function, NodeView nodeView)
 
             publishCustomNodeItem.Command = nodeView.ViewModel.DynamoViewModel.PublishSelectedNodesCommand;
             publishCustomNodeItem.CommandParameter = functionNodeModel;
-
-            nodeView.UpdateLayout();
```

Comment:
Was this moved to other places based on your PR description?

Reply:
Updated the PR description